cryptopals/set1/challenge1/submitted_solutions/hex_to_base64.py

```diff
@@ -0,0 +1,61 @@
+import codecs
+
+
+hex_test = "49276d206b696c6c696e6720796f757220627261696e206c696b65206120706f69736f6e6f7573206d757368726f6f6d"
+
+# 010010 010010 0111
+
+# 010010 010010 0111
+
+base64_test = "SSdtIGtpbGxpbmcgeW91ciBicmFpbiBsaWtlIGEgcG9pc29ub3VzIG11c2hyb29t\n"
+
+## b64 = codecs.encode(codecs.decode(hex, 'hex'), 'base64').decode()
+
+## print (b64)
+
+
+def test( hex):
+  return codecs.encode(codecs.decode(hex, 'hex'), 'base64').decode()
+
+
+
+def hex_to_base64 ( hex): 
```

Comment:
nice. you actually got some code in for the inner workings of the conversion. 👏

Reply:
this is awesome!